recipes/cuda-cudart/meta.yaml

```
@@ -0,0 +1,146 @@
+{% set name = "cuda-cudart" %}
+{% set version = "12.0.107" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-ppc64le" %}  # [ppc64le]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set platform = "windows-x86_64" %}  # [win]
+{% set extension = "tar.xz" %}  # [not win]
+{% set extension = "zip" %}  # [win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/cuda_cudart/{{ platform }}/cuda_cudart-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: c0e16e9593a69afe4997ae96a8b024e751ebce0254523fd969b08bb028582d23  # [linux64]
+  sha256: d785fd613e35e3761fdb0016a31a56417f570605cbfb9a93ca9dc68be6789188  # [ppc64le]
+  sha256: 5de248c6c203f6c91ce34256bbfc028bc87c1b0c552f9f51edb65e60ad665f63  # [aarch64]
+  sha256: 1cd8516feee068c6d718453e76b0dfcbee27844a35d4f3aa608a3d316190309c  # [win]
+
+build:
+  number: 0
+  skip: true  # [osx]
+  run_exports:                # [linux]
+    strong:                   # [linux]
+      - __glibc >=2.17        # [linux]
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+  host:
+    - sysroot_{{ target_platform }} 2.17   # [linux]
+    - __glibc >=2.17                       # [linux]
+  run:
+    - sysroot_{{ target_platform }} 2.17   # [linux]
+    - __glibc >=2.17                       # [linux]
+
+outputs:
+  - name: cuda-cudart
+    test:
+      commands:
+        - test -f $PREFIX/lib/libcudart.so.{{ version }}     # [linux]
+        - if not exist %LIBRARY_BIN%\cudart64_*.dll exit 1   # [win]
+    files:
+      - lib/libcu*.so.*           # [linux]
+      - targets/*/lib/*.so.*      # [linux]
+      - Library\bin\cudart*.dll   # [win]
+
+  - name: cuda-cudart-dev
+    run_exports:
+      weak:
+        - {{ pin_subpackage("cuda-cudart", max_pin='x') }}
+    requirements:
+      run:
+        - cuda-cudart >={{ version }}
+        - cuda-cudart-static >={{ version }}
+        - cuda-cudart-dev_{{ target_platform }} =={{ version }}
+        - cuda-cccl >=12.0,<13
+    test:
+      commands:
+        - test -d $PREFIX/lib/pkgconfig  # [linux]
+    files:             # [linux]
+      - lib/*.so       # [linux]
+      - include        # [linux]
+      - lib/pkgconfig  # [linux]
+
+  - name: cuda-cudart-dev_{{ target_platform }}
+    run_exports:
+      weak:
+        - {{ pin_subpackage("cuda-cudart", max_pin='x') }}
+    build:
+      noarch: generic
+    requirements:
+      run:
+        - cuda-cudart-static_{{ target_platform }} >={{ version }}
```

Per https://github.com/conda-forge/cuda-version-feedstock/issues/1#issuecomment-1462998398, we should pin to `cuda-version`. (Do we pin that on every package, or just the top level `cuda-cudart`? This suggestion applies similarly below, if we intend to pin every subpackage.)
```suggestion
        - cuda-cudart-static_{{ target_platform }} >={{ version }}
        - cuda-version =12.0
```